opta/core/terraform.py

```diff
@@ -117,10 +118,10 @@ def validate_version(cls) -> None:
                 f"Invalid terraform version {current_version} -- must be at least {MIN_TERRAFORM_VERSION}"
             )
 
-        if current_parsed >= version.parse(MAX_TERRAFORM_VERSION):
-            raise UserErrors(
-                f"Invalid terraform version {current_version} -- must be less than {MAX_TERRAFORM_VERSION}"
-            )
+        # if current_parsed >= version.parse(MAX_TERRAFORM_VERSION):
```

Comment:
if we not use some code, we can remove it to stay lean.
There is always github history to find previous versions.